src/testing/data/linkedaccounts-2022-10-23.json

```diff
@@ -0,0 +1,157 @@
+            "transferStatus": 2,
+            "lockable": false,
+            "state": 0,
+            "dismantlePermission": 0,
+            "isWrapper": false
+          },
+          "TigerSteam": {
+            "itemHash": 3147280338,
+            "quantity": 0,
+            "bindStatus": 0,
+            "location": 1,
+            "bucketHash": 130233231,
+            "transferStatus": 2,
+            "lockable": false,
+            "state": 0,
+            "dismantlePermission": 0,
+            "isWrapper": false
+          },
+          "BungieNext": {
+            "itemHash": 3147280338,
+            "quantity": 0,
+            "bindStatus": 0,
+            "location": 1,
+            "bucketHash": 130233230,
+            "transferStatus": 2,
+            "lockable": false,
+            "state": 0,
+            "dismantlePermission": 0,
+            "isWrapper": false
+          },
+          "TigerEgs": {
+            "itemHash": 3147280338,
+            "quantity": 0,
+            "bindStatus": 0,
+            "location": 1,
+            "bucketHash": 130233229,
+            "transferStatus": 2,
+            "lockable": false,
+            "state": 0,
+            "dismantlePermission": 0,
+            "isWrapper": false
+          }
+        }
+      },
+      "crossSaveOverride": 2,
+      "applicableMembershipTypes": [1, 3, 5, 2],
+      "isPublic": false,
+      "membershipType": 2,
+      "membershipId": "4611686018433092312",
+      "displayName": "VidBoi-BMC",
+      "bungieGlobalDisplayName": "VidBoi",
+      "bungieGlobalDisplayNameCode": 9226
+    }
+  ],
+  "bnetMembership": {
+    "supplementalDisplayName": "VidBoi#9226",
+    "iconPath": "/img/profile/avatars/cc25.jpg",
+    "crossSaveOverride": 0,
+    "isPublic": false,
+    "membershipType": 254,
+    "membershipId": "7094",
+    "displayName": "VidBoi",
+    "bungieGlobalDisplayName": "VidBoi",
+    "bungieGlobalDisplayNameCode": 9226
+  },
+  "profilesWithErrors": [
+    {
+      "errorCode": 1601,
+      "infoCard": {
+        "crossSaveOverride": 2,
+        "applicableMembershipTypes": [],
+        "isPublic": true,
+        "membershipType": 1,
+        "membershipId": "4611686018429726245",
+        "displayName": "Vid Boi",
+        "bungieGlobalDisplayName": "VidBoi",
+        "bungieGlobalDisplayNameCode": 9226
+      }
+    },
+    {
+      "errorCode": 1601,
+      "infoCard": {
+        "crossSaveOverride": 2,
+        "applicableMembershipTypes": [],
+        "isPublic": true,
+        "membershipType": 3,
+        "membershipId": "4611686018509956320",
+        "displayName": "VidBoi",
+        "bungieGlobalDisplayName": "VidBoi",
+        "bungieGlobalDisplayNameCode": 9226
+      }
+    },
+    {
+      "errorCode": 1601,
+      "infoCard": {
+        "crossSaveOverride": 2,
+        "applicableMembershipTypes": [],
+        "isPublic": true,
+        "membershipType": 5,
+        "membershipId": "4611686018509956086",
```

Comment:
would you want to obscure your membership/display names?

Reply:
I'm guessing it's just easier to export & test with real data but was just a thought

Reply:
It doesn't matter to me